includes/content-registration/custom-post-types-registration.php

```diff
@@ -265,7 +265,7 @@ function generate_custom_post_type_args( array $args ): array {
 
 	$singular = $args['singular'];
 	$plural   = $args['plural'];
-	$icon     = require ATLAS_CONTENT_MODELER_DIR . '/includes/settings/views/admin-entry-icon.php';
+	$icon     = include_once ATLAS_CONTENT_MODELER_INCLUDES_DIR . 'settings/views/admin-entry-icon.php';
```

Comment:
This is a case where it has to be `include` instead of `include_once`, otherwise we only get the custom admin menu icon for the first CPT (cats in the screenshot below) and not subsequent ones (sloths):

<img width="119" alt="Screenshot 2021-06-23 at 17 43 14" src="https://user-images.githubusercontent.com/647669/123127496-9eb0fe00-d44a-11eb-9a0e-7b970a4a168e.png">

```suggestion
	$icon     = include ATLAS_CONTENT_MODELER_INCLUDES_DIR . 'settings/views/admin-entry-icon.php';
```